.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+.DS_Store
```

Comment:
Only artefacts created by a project should be included in the project's __.gitignore__, in my view. Otherwise every project's __.gitignore__ will eventually contain hundreds of lines of noise relating to various operating systems and text editors. It's better for macOS users to add `.DS_Store` to their [global __.gitignore__][1]. :)


[1]: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore